content/docs/faq-versioning.md

```diff
@@ -1,48 +1,48 @@
 
-Development warnings won't affect the runtime behavior of your app. That way, you can feel confident that your app will behave the same way between the development and production builds -- the only differences are that the production build won't log the warnings and that it is more efficient. (If you ever notice otherwise, please file an issue.)
+Las advertencias de desarrollo no afectarán el comportamiento en tiempo de ejecución de tu aplicación. De esa manera, puedes sentir la confianza de que tu aplicación se comportará de la misma manera entre la versión de desarrollo y la versión de producción -- las únicas diferencias son que la versión de producción no registrará las advertencias y es más eficiente. (Si alguna vez observa lo contrario, por favor presentar un *issue*.)
 
-### What Counts as a Breaking Change? {#what-counts-as-a-breaking-change}
+### ¿Qué cuenta como un cambio de ruptura? {#what-counts-as-a-breaking-change}
 
-In general, we *don't* bump the major version number for changes to:
+En general, *no publicamos* una versión mayor por cambios a:
 
-* **Development warnings.** Since these don't affect production behavior, we may add new warnings or modify existing warnings in between major versions. In fact, this is what allows us to reliably warn about upcoming breaking changes.
-* **APIs starting with `unstable_`.** These are provided as experimental features whose APIs we are not yet confident in. By releasing these with an `unstable_` prefix, we can iterate faster and get to a stable API sooner.
-* **Alpha and canary versions of React.** We provide alpha versions of React as a way to test new features early, but we need the flexibility to make changes based on what we learn in the alpha period. If you use these versions, note that APIs may change before the stable release.
-* **Undocumented APIs and internal data structures.** If you access internal property names like `__SECRET_INTERNALS_DO_NOT_USE_OR_YOU_WILL_BE_FIRED` or `__reactInternalInstance$uk43rzhitjg`, there is no warranty.  You are on your own.
+* **Advertencias de desarrollo.** Dado que no afectan al comportamiento de producción, podemos añadir nuevas advertencias o modificar advertencias existentes entre las versiones mayores. De hecho, esto es lo que nos permite advertir de forma fiable sobre los próximos cambios de ruptura.
+* **APIs que comienzan con `unstable_`.** Estos ofrecen funcionalidades experimentales sobre cuyos APIs todavía no estamos seguros. Al publicar esto con un prefijo `unstable_`, podemos iterar más rápido y llegar a un API estable lo antes posible.
+* **Versiones alfa y canarias de React.**  Proporcionamos versiones alfa de React como una manera de probar nuevas características temprano, pero necesitamos la flexibilidad para hacer cambios basados en lo que aprendemos en el período alfa.
```

Comment:
Es común ver versiones alfa bajo ese nombre, pero `canary` traducido a `canarias` no tanto. Creo que esto se podría dejar como el original.

```suggestion
* **Versiones alfa y *canary* de React.**  Proporcionamos versiones alfa de React como una manera de probar nuevas características con antelación, pero necesitamos la flexibilidad para hacer cambios basados en lo que aprendemos en el período alfa.
```

Adicionalmente, falta traducir: `If you use these versions, note that APIs may change before the stable release.`